### ISSUE TITLE
result of GradientBoostingRegressor

### ISSUE THREAD

**Issue by MingyuLi19910814** [srikhetramohanty/Data-Science-Portfolio#2]
Firstly of all, thank you very much for the code. It's very clear and easy to understand.
Just a little comment, the rmse of sklearn GradientBoostingRegressor is higher than the implemented tree comes from the learning rate. In sklearn, the default learning rate is set as 1.0, while in your implementation, the learning rate is 0.01. If you set the learning rate of the sklearn as 0.01, it would generate the same rmse.
